### PR TITLE
update README with rebranding

### PR DESCRIPTION
**Pull request by 0xpulkit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ Please check the [IDEN3MESSAGE_PARSER.md](IDEN3MESSAGE_PARSER.md) file for more
 
 ## Description
 
-This is a Flutter plugin for the [Polygon ID SDK](https://polygon.technology/polygon-id), which allows you to integrate Polygon ID identity system into your Flutter apps.
+This is a Flutter plugin for the [Privado ID SDK](https://docs.privado.id/docs/category/wallet-sdk), which allows you to integrate Privado ID identity system into your Flutter apps.
 
-Please see the [example app](https://github.com/iden3/polygonid-flutter-sdk/tree/develop/example) included in the repository and follow the [Polygon ID Wallet SDK Documentation](https://0xpolygonid.github.io/tutorials/wallet/wallet-sdk/polygonid-sdk/polygonid-sdk-overview/).
+Please see the [example app](https://github.com/iden3/polygonid-flutter-sdk/tree/develop/example) included in the repository and follow the [Privado ID Wallet SDK Documentation](https://0xpolygonid.github.io/tutorials/wallet/wallet-sdk/polygonid-sdk/polygonid-sdk-overview/).
```
Comment (5eeman) [inline README.md:21]: https://0xpolygonid.github.io/tutorials/wallet/wallet-sdk/polygonid-sdk/polygonid-sdk-overview/
This link is broken. Leads to 404.

File: README.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ See [below](#environment) for details about setting up the environment.
 Stream<DownloadInfo> stream =
         await PolygonIdSdk.I.proof.initCircuitsDownloadAndGetInfoStream;
 ```
-For more information on how to use the PolygonID Flutter SDK, please check the [example app](https://github.com/iden3/polygonid-flutter-sdk/tree/develop/example) included in the repository and follow the [Polygon ID Wallet SDK Documentation](https://0xpolygonid.github.io/tutorials/wallet/wallet-sdk/polygonid-sdk/polygonid-sdk-overview/)
+For more information on how to use the Privado ID Flutter SDK, please check the [example app](https://github.com/iden3/polygonid-flutter-sdk/tree/develop/example) included in the repository and follow the [Privado ID Wallet SDK Documentation](https://0xpolygonid.github.io/tutorials/wallet/wallet-sdk/polygonid-sdk/polygonid-sdk-overview/)
```
Comment (5eeman) [inline README.md:87]: Same. Broken.

**Review by 5eeman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (0xpulkit) [reply README.md]: Changed it

File: README.md (hunk 2 above)
Comment (0xpulkit) [reply README.md]: Changed

**Review by 5eeman — APPROVED** [review]

**Pull request merged**